### PR TITLE
Add error logs/tags in asynclistener on error handler

### LINKED ISSUE #36: Log exception from Async handlers
https://github.com/opentracing-contrib/java-web-servlet-filter/blob/master/opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java#L197

### PR DESCRIPTION
**Pull request by pavolloffay**
Resolves #36 

Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

**Comment by pavolloffay** [comment]
Unfortunately I wasn't able to test this. Here is the test https://github.com/pavolloffay/java-web-servlet-filter/compare/async-exception-clean...pavolloffay:async-exception-clean-test?expand=1. The error callback is not being invoked.

However I have tested this with spring-web on SB and callback with the exception is invoked. However MVC does not call these callbacks :/

File: opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java [hunk 1]
```diff
@@ -205,6 +205,14 @@ public void onTimeout(AsyncEvent event) throws IOException {
 
                         @Override
                         public void onError(AsyncEvent event) throws IOException {
+                            HttpServletRequest httpRequest = (HttpServletRequest) event.getSuppliedRequest();
+                            HttpServletResponse httpResponse = (HttpServletResponse) event.getSuppliedResponse();
+                            for (ServletFilterSpanDecorator spanDecorator: spanDecorators) {
+                                spanDecorator.onError(httpRequest,
```
Comment (jpkrohling) [inline opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java:211]: Does it fit all in one line?

**Review by jpkrohling — APPROVED** [review]
Looks sane to me, but without a test, it's hard to tell it works :)

File: opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java (hunk 1 above)
Comment (pavolloffay) [reply opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java:211]: There is no code style check :)

I use this to be consistent with other handlers.

**Pull request merged**